bears/yaml/TravisLintBear.py

```diff
@@ -24,6 +26,29 @@ class TravisLintBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
+    CHECK_CONNECTION_URL = 'https://travis-ci.org/'
+
+    @classmethod
+    def check_prerequisites(cls):  # pragma: no cover
```

Comment:
I had actually written tests for the new feature but this line becomes essential for skipping the `base_check`. However, I've updated it. :wink: